.packit.yaml

```diff
@@ -3,33 +3,67 @@
 # https://packit.dev/docs/configuration/
 
 downstream_package_name: python-podman
-specfile_path: rpm/python-podman.spec
 upstream_tag_template: v{version}
 
+packages:
+  python-podman-fedora:
+    pkg_tool: fedpkg
+    specfile_path: rpm/python-podman.spec
+  python-podman-centos:
+    pkg_tool: centpkg
+    specfile_path: rpm/python-podman.spec
+  python-podman-rhel:
+    specfile_path: rpm/python-podman.spec
+
 srpm_build_deps:
   - make
 
 jobs:
+  # Copr builds for Fedora
   - job: copr_build
     trigger: pull_request
+    packages: [python-podman-fedora]
     targets:
       - fedora-all
-      - centos-stream-8
+
+  # Copr builds for CentOS Stream
+  - job: copr_build
+    trigger: pull_request
+    packages: [python-podman-centos]
+    targets:
+      - centos-stream-10
       - centos-stream-9
 
+  # Copr builds for RHEL
+  - job: copr_build
+    trigger: pull_request
+    packages: [python-podman-rhel]
+    targets:
+      - epel-9
+
   # Run on commit to main branch
   - job: copr_build
     trigger: commit
     branch: main
     owner: rhcontainerbot
     project: podman-next
 
+  # Downstream sync for Fedora
   - job: propose_downstream
     trigger: release
+    packages: [python-podman-fedora]
     update_release: false
     dist_git_branches:
       - fedora-all
 
+  # Downstream sync for CentOS Stream
+  - job: propose_downstream
+    trigger: release
+    packages: [python-podman-centos]
+    update_release: false
+    dist_git_branches:
+      - c10s
```

Comment:
This might be a trivial question, that it's simply a missing context from my side. Why not c9s too?

Reply:
Yes, I'd like to have c9s too, but the trouble in c9s is that the buildroot doesn't have rpmautospec yet so we'll need additional gymnastics to handle changelog which I'd rather avoid. I've filed a jira to get rpmautospec included and will enable c9s soon as it's resolved.

Reply:
I see, thanks. Would it make sense to include the JIRA issue in a TODO comment in the code so we can keep track of it in the packit file?

Other than that the PR looks good.

Reply:
done. PTAL

Reply:
thanks, LGTM